### DISCUSSION TITLE
Plant-it Floracodex Integration (CasaOS)

### DISCUSSION THREAD

**Discussion by satvikknischal** [MDeLuise/plant-it#373] [Q&A]
Hi, full disclosure I am extremely new to self hosting, just came across the Plant-it app and thought it would be of great help to my mother who does alot of plantation. 

I wanted to integrate the Floracodex API key with the app but can't seem to figure out how to do that. I basically installed Plant-it via the CasaOS Big Bear App Store, and there are no instruction on how to integrate Floracodex (atleast none that I could find). Any help would be greatly appreciated.

Thankyou

**Comment by MateoGreil** [comment]
Hi,

This is not clearly written, but you can guess here : https://docs.plant-it.org/0.9.0/server-installation/#integration-with-floracodex
That you need to add to the env var of your docker the floracodex key as `FLORACODEX_KEY`.

So in your CasaOS app setting, go to the plant-it and add an environment variable.
Here is mine ([I'm using the Cp0204 AppStore one](https://github.com/Cp0204/CasaOS-AppStore-Play/tree/50df0a6b3744fadf3442325f30482be37482dc77/Apps/plant-it), but it should be the same for the BigBear one):
(at the PUT IT HERE)
![image](https://github.com/user-attachments/assets/ec9fa5d2-cd76-46af-9d64-65fb1a0b6493)
